apps/api/migrations/081_add_apy_snapshot_anomaly_flag.up.sql

```
@@ -0,0 +1,5 @@
+ALTER TABLE apy_snapshots
+    ADD COLUMN flagged BOOLEAN NOT NULL DEFAULT FALSE,
+    ADD COLUMN flag_reason TEXT;
+
+CREATE INDEX idx_apy_snapshots_flagged ON apy_snapshots (protocol_slug, captured_at) WHERE flagged;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)